alphastats/plots/VolcanoPlot.py

```diff
@@ -152,7 +175,7 @@ def _sam_calculate_fdr_line(self):
         )
 
     @lru_cache(maxsize=20)
-    def _sam(self):
+    def _sam(self):  # TODO duplicated?
```

Comment:
Only the last two lines differ from the function in DifferentialExpressionAnalysis